ethereum/p2p/src/main/java/org/hyperledger/besu/ethereum/p2p/discovery/PeerDiscoveryAgent.java

```diff
@@ -84,13 +84,13 @@
   protected final Subscribers<PeerBondedObserver> peerBondedObservers = Subscribers.create();
 
   public PeerDiscoveryAgent(
-      final SECP256K1.KeyPair keyPair,
+      final NodeKey nodeKey,
       final DiscoveryConfiguration config,
       final PeerPermissions peerPermissions,
       final NatService natService,
       final MetricsSystem metricsSystem) {
     this.metricsSystem = metricsSystem;
-    checkArgument(keyPair != null, "keypair cannot be null");
+    checkArgument(nodeKey != null, "cryptoOps cannot be null");
```

Comment:
nit: `cryptoOps` should be `nodeKey` in exception message.

Reply:
done